config.toml

```diff
@@ -1,6 +1,6 @@
 # sample spacemesh config file
 # use the config flag to start a node with a config file.
-# e.g $./go-spacemash --config ./config.toml
+# e.g $./go-spacemesh --config ./config.toml
```

Comment:
changes to this file will conflict with your other PR...

Reply:
appreciate the heads up! I've already rebased on the latest develop branch so I think we are good here!